monocle/worker.py

```diff
@@ -780,8 +780,43 @@ def travel_speed(self, point):
                         if not cooldown or time() > cooldown / 1000:
                             spinning = LOOP.create_task(self.spin_pokestop(pokestop))
                 else:
-                    DB_PROC.add(self.normalize_gym(fort))
 
+                    fort['name'] = ""
+                    if not self.normalize_gym(fort) in FORT_CACHE:
+                        request = self.api.create_request()
+                        request.get_gym_details(gym_id=fort['id'],
+                                                player_latitude=latitude,
+                                                player_longitude=longitude,
+                                                gym_latitude=fort['latitude'],
+                                                gym_longitude=fort['longitude']
+                                                )
+                        responses = await self.call(request)
+
+
+
+                        result = responses.get('GET_GYM_DETAILS', {}).get('status', 0)
```

Comment:
check on result, not status
https://github.com/AeonLucid/POGOProtos/blob/master/src/POGOProtos/Networking/Responses/GetGymDetailsResponse.proto